homework/blackjack/card.py

```diff
@@ -0,0 +1,64 @@
+from xprint import xPrint, printStack
+
+
+class Card():
```

Comment:
class Card(object): # read about "old and new classes" in https://docs.python.org/3/whatsnew/2.2.html